src/App.tsx

```diff
@@ -14,6 +38,124 @@ export const App: React.FC = () => {
     }
   }, []);
 
+  const getTodosFromApi = async (userId: number) => {
+    try {
+      const response = await getTodos(userId);
+
+      setTodos(response);
+      setIsAdding(null);
+      setIsEditing(null);
+    } catch (error) {
+      throw new Error('Error on loading todos');
+    }
+  };
+
+  const createTodo = async (event: React.FormEvent) => {
+    event.preventDefault();
+
+    if (!inputTitle) {
+      setInputError(true);
+      setIsHidden(false);
+
+      return;
+    }
+
+    if (user) {
+      const newTodo = {
+        userId: user.id,
+        title: inputTitle,
+        completed: false,
+      };
+
+      setIsAdding(newTodo);
+
+      try {
+        await createTodos(newTodo);
+      } catch {
+        setAddError(true);
+        setIsHidden(false);
+        setIsAdding(null);
+      }
+
+      getTodosFromApi(user.id);
+      setInputTitle('');
+    }
+  };
+
+  const selectCompleted = async (todo: Todo) => {
```

Comment:
useCallback

Reply:
and again, this name is confusing

from method that called `selectCompleted` I expect that it will select completed todos for something, but in reality this method updating todos on server 

let's use more clear names